src/compute/host/host_device_queue.cpp

```diff
@@ -38,5 +38,10 @@ void host_device_queue::wait_until_completed() const
     // NO-OP
 }
 
+bool host_device_queue::is_idle() const noexcept
+{
+    return true;
+}
+
```

Comment:
Why always idle?

Reply:
Yes. Host is synchronous. That's why there is only a single queue.